sdk/provisioning/Azure.Provisioning/src/Resource.cs

```diff
@@ -59,6 +61,10 @@ internal void AddDependency(Resource resource)
         /// </summary>
         public IList<Parameter> Parameters { get; }
 
+        internal IList<Output> Outputs { get; }
```

Comment:
We recently took outputs off of a resource I am curious why we needed to add it back.

The reason we removed this was because an output is a Construct/Module concept.  A resource might have a property that is used in an output, but a Resource does not have an output.